### ISSUE TITLE
/install doesn't seem to exist

### ISSUE THREAD

**Issue by mrme44** [freescout-helpdesk/freescout#3845]
I'm trying to install freescout using the "Interactive installation bash-script " on a clean Ubuntu droplet from Digital Ocean. When  it finishes, it gives me a helpdesk URL of https://mywebsite.com/install, but when I go to /install I get a 404 error. I also tried /install.php and I got the same error. Looking at the repo, I don't see any files named install.

**Comment by Erudition** [comment]
`/install` doesn't exist as a file but in your `/public` folder (where all requests go) there should be an `install.php`.

I used Softaculous to install, which was broken, but I re-ran the installer by following the install guide: delete `.env`, delete `/storage/.installed`, and delete everything _inside_ `/bootstrap/cache` (but not the folder itself, or you have a crash).

Then navigating to `/install` got me the installer, where I was able to see all the passed checks (and re-enter my old `.env` file's settings).
